app/models/orphan.rb

```diff
@@ -66,6 +66,8 @@ class Orphan < ActiveRecord::Base
   accepts_nested_attributes_for :current_address, allow_destroy: true
   accepts_nested_attributes_for :original_address, allow_destroy: true
 
+  default_scope { includes(:partner, :orphan_sponsorship_status, :orphan_status, original_address: :province) }
```

Comment:
I don't think so, especially for our current situation.
- For a find that returns a single record the difference will be largely unnoticeable, even if the relationships weren't needed.
- For returning large row sets in the index view, the benefit gained by eager loading is dramatic.
- If there were any situations where the relationships weren't needed (I can't think of any off hand  where we wouldn't use something like pluck) we could override the default scope.
- Active admin uses the default scope for filters so for production as it stands at the moment I think it is the best choice for performant rendering.

If this irritates us post active admin I think we could consider other options, but for the moment I think it is our best choice. 
 
As for enums, the last time we talked about this we were holding off until active admin was out of the picture so we didn't have to amend code we were going to be letting go of. I think this is probably still the best course of action.